parcels/examples/tutorial_geospatial.ipynb

```diff
@@ -0,0 +1,1056 @@
+    "    \"\"\"Writes parcels trajectories to KML file.\n",
+    "    \n",
+    "    Converts the GeoDataFrame from the `parcels_to_geopandas` function into KML\n",
+    "    for use in Google Earth. Each particle trajectory is converted to a gx:Track item\n",
+    "    to include timestamp information in the path.\n",
+    "    \n",
+    "    Only uses the trajectory ID, time, lon, and lat variables.\n",
+    "    \n",
+    "    Parameters\n",
+    "    ----------\n",
+    "    gdf : gpd.GeoDataFrame\n",
+    "        GeoDataFrame as output from the `parcels_to_geopandas` function\n",
+    "        \n",
+    "    path : pathlike\n",
+    "        Path to save the KML to.\n",
+    "        \n",
+    "    rubber_ducks : bool\n",
+    "        Replace default particle marker with rubber duck icon.\n",
+    "    \n",
+    "    \n",
+    "    See also\n",
+    "    --------\n",
+    "    More on gx:Track in KML:\n",
+    "    https://developers.google.com/kml/documentation/kmlreference#gx:track\n",
+    "    \"\"\"\n",
+    "    # Define namspaces\n",
+    "    kml_ns = \"http://www.opengis.net/kml/2.2\"\n",
+    "    gx_ns = \"http://www.google.com/kml/ext/2.2\"\n",
+    "\n",
+    "    kml_out = etree.Element(\"{%s}kml\" % kml_ns, nsmap={None: kml_ns, \"gx\": gx_ns})\n",
+    "    document = etree.SubElement(kml_out, \"Document\", id=\"1\", name=\"Particle trajectories\")\n",
+    "        \n",
+    "    # Custom icon styling\n",
+    "    if rubber_ducks:\n",
+    "        icon_styling = etree.fromstring(\n",
+    "            f\"\"\"<Style id=\"iconStyle\">\n",
+    "                  <IconStyle>\n",
+    "                    <scale>0.8</scale>\n",
+    "                    <Icon>\n",
+    "                      <href>https://icons.iconarchive.com/icons/thesquid.ink/free-flat-sample/256/rubber-duck-icon.png</href>\n",
+    "                    </Icon>\n",
+    "                  </IconStyle>\n",
+    "                </Style>\n",
+    "            \"\"\"\n",
+    "        )\n",
+    "        document.append(icon_styling)\n",
+    "    \n",
+    "    # Generating gx:Track items\n",
+    "    for trajectory_idx, trajectory_gdf in gdf.groupby(\"trajectory\"):\n",
+    "        trajectory_gdf = trajectory_gdf.sort_values(\"obs\")\n",
+    "\n",
+    "        placemark = etree.SubElement(document, \"Placemark\")\n",
+    "        name_element = etree.SubElement(placemark, \"name\")\n",
+    "        name_element.text = str(trajectory_idx)\n",
+    "        \n",
+    "        # Link custom icon styling\n",
+    "        if rubber_ducks:\n",
+    "            style_url = etree.SubElement(placemark, \"styleUrl\")\n",
+    "            style_url.text = \"#iconStyle\"\n",
+    "        \n",
+    "        gx_track = etree.SubElement(placemark, \"{%s}Track\" % gx_ns)\n",
+    "        altitude_mode = etree.SubElement(gx_track, \"{%s}altitudeMode\" % gx_ns, text=\"clampToGround\")\n",
```

Comment:
`altitude_mode` is not used in the function?

Reply:
Good catch. `etree.SubElement` works in place as well as returns a reference. In this case the reference wasn't used.
```suggestion
    "        etree.SubElement(gx_track, \"{%s}altitudeMode\" % gx_ns, text=\"clampToGround\")\n",
```